apps/api/src/mail/services/interface.service.ts

```diff
@@ -5,11 +5,12 @@ export interface IMailService {
 
   sendEmailChangedOtp(email: string, otp: string): Promise<void>
 
-  workspaceInvitationMailForUsers(
+  invitedToWorkspace(
     email: string,
-    workspace: string,
-    actionUrl: string,
+    projectName: string,
```

Comment:
Not sure why you made this change. Considering the fact that people won't be invited to projects but workspaces.